src/js/gi/components/search/SearchResult.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import * as actions from '../../actions';
+
+import { Link } from 'react-router';
+import { estimatedBenefits } from '../../selectors/estimator';
+
+export class SearchResult extends React.Component {
+
+  cautionFlag() {
+    if (!this.props.cautionFlag) {
+      return null;
+    }
+    return (
+      <div className="caution-flag">
+        <i className="fa fa-warning"></i>
+        Caution
+      </div>
+    );
+  }
+
+  place() {
+    const domestic = <span>{this.props.city}, {this.props.state}</span>;
+    const foreign = <span>{this.props.city}, {this.props.country}</span>;
+    return (this.props.country === 'usa' ? domestic : foreign);
```

Comment:
You can avoid extra JSX processing here, though it's not a big deal in this case because it's very simple. Just as an example, with the following, you end up only processing one JSX fragment.

```
return (
  <span>
    {this.props.city}, {this.props.country === 'usa' ? this.props.state : this.props.country}
  </span>
);
```